### PR TITLE
Awesome books with ES6

### PR DESCRIPTION
**Pull request by AnicetFantomas**
in this pull request we performed :

- Use ES6 modules to write modular JavaScript.
- Learn how to use proper ES6 syntax.
- Understand improvements that ES6 brings to JavaScript.
- Use npm as software packages system.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import UiBook from './UiBook.js';
+import UiBook from './modules/UiBook.js';
+import { DateTime } from './modules/luxon.js';
```
Comment (thecodechaser) [inline main.js:3]: - [ ] For this project you are supposed to name the entry point for your JavaScript code called `index.js`. Please change this file name to `index.js` from `main.js`. 

### See the image for reference

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @AnicetFantomas👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
- [ ] For this project you are supposed to name the styles file called `index.css`. Please change your styles file name to `index.css` from `style.css`. 

### See the image for reference

image

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
         </nav>
     </header>
     <main>
-        <span class="date">Date:</span>
+        <div class="date"></div>
```
Comment (BenMukebo) [inline index.html:24]: - [ ] Great job so far 👏🏻  However as required by the previous reviewer, Kindly change your style file name, rename it from `style.css` to `index.css` see the project requirement  ⬇️

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@AnicetFantomas,**

Great job on making changes required by the previous reviewer 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Githubflow was correctly used.  ✔️   
- Good arrow syntax for functions implementation. ✔️ 

 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly, it will be great to add instructions on how to run the project in your README file. This includes `npm install` , `npm start` etc

Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, @AnicetFantomas.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**